axlearn/common/attention.py

```diff
@@ -1108,6 +1110,28 @@ def forward(self, x: Tensor) -> Tensor:
         return (x * scale).astype(x.dtype)
 
 
+ScaleFn = Callable[[int], float]  # A function mapping per_head_dim to a scale.
+
+
+def constant_scale_config(value: float) -> InstantiableConfig[ScaleFn]:
+    """A config for a constant scale function for `MultiheadAttention`.
+
+    Args:
+        value: The value to scale by.
+
+    Example:
+        `query_scale = config_for_function(constant_scale).set(value=0.01)`
+
+    Returns:
+        A config that scales by `value`.
+    """
+
+    def constant_function(_: float, value: float) -> float:
```

Comment:
```suggestion
    def constant_function(per_head_dim: int, value: float) -> float:
```

Reply:
Oops, it got automerged before I made this change.

However, making this change would cause a pytype failure due to the unused argument. Should I make the change and tell pytype to ignore it?